tasks/managed/create-pyxis-image/tests/test-create-pyxis-image-multi-containerimages-multi-arch.yaml

```diff
@@ -219,6 +219,7 @@ spec:
                 cat "$(params.dataDir)/mock_create_container_image.txt"
                 exit 1
               fi
+              cat "$(params.dataDir)/mock_create_container_image.txt"
```

Comment:
not sure why this is added. It is already printed if the length isn't as expected

Reply:
leftover from debugging. removed